js/__internal/grids/grid_core/filter/m_filter_row.ts

```diff
@@ -673,10 +673,13 @@ const ColumnHeadersViewFilterRowExtender = (function () {
     },
 
     _handleDataChanged(e) {
```

Comment:
Minor comment.

Let's refactor this function a little
Something like this:
```suggestion
    _handleDataChanged(e) {
      this.callBase.apply(this, arguments);
      
      const { filtering, fullReload } = e.operationTypes;
      if (!filtering && !fullReload) {
        return;
      }
      
       const dataSource = this._dataController?.dataSource?.();
       const lastLoadOptions = dataSource?.lastLoadOptions?.();
       const loadOptions = filtering ?? lastLoadOptions?.filter;
       this.updateLookupDataSource(loadOptions);
    }
```